myproject/blog/models.py

```diff
@@ -1,24 +1,26 @@
 from django.db import models
 from django.utils import timezone
+from django.contrib.auth.models import User
 
 class BaseModel(models.Model):
-    created_at = models.DateTimeField(auto_now_add=True)
+    created_at = models.DateTimeField(default=timezone.now, editable=False)
     updated_at = models.DateTimeField(auto_now=True)
 
     class Meta:
         abstract = True
         db_table = "base_model"
```

Comment:
There will be no db table creation for the abstract class. Relearn the concept again from Django docs.